python/dronin-getconfig

```diff
@@ -57,15 +57,18 @@ def main():
     wait_till_less_than(0)
 
     missing = []
+    sobjs = []
     for s in settings_objects:
         val = tStream.last_values.get(s)
         if val is not None:
-            print(val)
+            sobjs.append(val)
         else:
             missing.append(s._name)
 
-    for s in missing:
-        print("No instance of %s" % (s))
+    #for s in missing:
+    #    print("No instance of %s" % (s))
```

Comment:
We should use python's excellent logging class to do this stuff, then it's easy to switch on debug type stuff when needed.

Reply:
Yah, I've got that on my todo list for a big cleanup... but I'm trying to sprint towards integration tests and I kinda hate the python logging stuff (it definitely takes thought on how to instantiate it right in a framework like this etc)